internal/terraform/lang/utils.go

```diff
@@ -18,18 +18,18 @@ func PosInLabels(b *hclsyntax.Block, pos hcl.Pos) bool {
 	return false
 }
 
-func prefixAtPos(looker TokenAtPosLooker, pos hcl.Pos) string {
+func prefixAtPos(looker TokenAtPosLooker, pos hcl.Pos) (string, *hcl.Range) {
 	token, err := looker.TokenAtPosition(pos)
 	if err != nil {
-		return ""
+		return "", nil
 	}
 
 	switch token.Type {
 	case hclsyntax.TokenIdent, hclsyntax.TokenQuotedLit, hclsyntax.TokenStringLit:
-		return string(token.Bytes[:pos.Byte-token.Range.Start.Byte])
+		return string(token.Bytes[:pos.Byte-token.Range.Start.Byte]), token.Range.Ptr()
```

Comment:
this is the range for prefix, so we should set the right end pos

Reply:
maybe it depends on how we handle an attribute re-completion, it's OK to override the whole word

Reply:
Yeah, that's a fair point. This is actually by design, it's just confusing that we're mixing prefix (left part of the token) and range of the whole token in one function.

I'll have a think about it.

Reply:
I can't come up with a better pattern, so I'm going to merge it as is, but I'm open to ideas and PRs to address this in the future.